membrane_curvature/tests/test_membrane_curvature.py

```diff
@@ -2,6 +2,7 @@
 Unit and regression test for the membrane_curvature package.
 """
 
+from numpy.lib.stride_tricks import DummyArray
```

Comment:
What's this for?

Reply:
Sorry, I have no idea how that line ended up there.